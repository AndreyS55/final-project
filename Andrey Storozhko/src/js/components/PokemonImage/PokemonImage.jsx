import React from 'react';
import noImage from '../../../img/no-image.png';

import styles from './PokemonImage.scss';

const PokemonImage = (props) => {
  const imgPath = props.id < 721 ? require(`../../../../pokemons/${props.id}.png`) : noImage;
  return <img src={imgPath} className={styles.pokemonImage} alt='Pokemon avatar' onClick={props.handleOpen}/>
};

export default PokemonImage;