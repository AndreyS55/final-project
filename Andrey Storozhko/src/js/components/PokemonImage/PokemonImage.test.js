import React from 'react';
import PokemonImage from './PokemonImage';

it('should render a PokemonImage', () => {
  const wrapper = shallow(<PokemonImage />);
  expect(wrapper).toMatchSnapshot();
});
