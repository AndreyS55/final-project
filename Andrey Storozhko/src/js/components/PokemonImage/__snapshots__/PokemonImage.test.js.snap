// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a PokemonImage 1`] = `
<img
  alt="Pokemon avatar"
  className="pokemonImage"
  src="test-file-stub"
/>
`;
